stamina-core/src/test/scala/stamina/migrations/MigratorSpec.scala

```diff
@@ -0,0 +1,85 @@
+package stamina.migrations
+
+import stamina._
+
+class MigratorSpec extends StaminaSpec {
+
+  val migrator = from[String, V1]
```

Comment:
This test suits to scalacheck, do you think I can add scalacheck as dependency and rewrite this test?